test/bench/report_generator.py

```diff
@@ -169,6 +169,19 @@ def makeSummaryGraph(outputDirectory, summary):
 def splitTagSemVer(tag):
     return list(filter(None, semVerRegex.split(tag)))
 
+# if the string is a number, return it. Otherwise convert
+# the string to a numeric value that sorts alphabetically
+# Assumes characters have a max value of 255.
+def numericValueOf(s):
+    try:
+        return int(s)
+    except ValueError:
+        accumulate = 0
+        ts = s[:min(7, len(s))]
```

Comment:
Why min with 7 rather than using the whole s?

Reply:
More than 7 has the possibility to overflow 64 bits, and python throws this : `parse error while processing semver of v6.0.18-327-g065306752 Python int too large to convert to C long`. I've added a comment about it.

Reply:
How old is the python version we are using? That should have been fixed in python 2.2 from 2001! https://docs.python.org/2/whatsnew/2.2.html#pep-237-unifying-long-integers-and-integers

Reply:
Whichever stock python comes with [ubuntu 18.04](https://github.com/realm/realm-core/blob/v10/Dockerfile#L1) which appears to be python 2.7. Not sure about what cases exactly cause that exception, but even if the value rolled over that would cause incorrect sorting. I'd love to upgrade our performance scripts to python 3 at some point. But if I did that I would probably also want to use a different graphing library so it could turn into a sizeable task.